functions.php

```diff
@@ -123,6 +123,17 @@ function process_signins_resultset($signins) {
     return $signins_return;
 }
 
+function signins_n_days_ago($daysAgo) {
+    $oneDayInMS = 24 * 60 * 60 * 100
```

Comment:
This should be 24 * 60 * 60 * 1000, or just 86400 * 1000.


Also you forgot a semicolon :)